### PR TITLE
Document libxml changes for PHP 8.4

### PR DESCRIPTION
**Pull request by nielsdos**

### CODE REVIEW

File: reference/libxml/functions/libxml-set-streams-context.xml [hunk 1]
```diff
@@ -42,6 +42,39 @@
   </para>
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <para>
+   Throws a <classname>TypeError</classname> when a non-stream resource is
+   passed to <parameter>context</parameter>.
+  </para>
```
Comment (Girgias) [inline reference/libxml/functions/libxml-set-streams-context.xml]: simpara :)

File: reference/libxml/functions/libxml-set-streams-context.xml [hunk 2]
```diff
@@ -42,6 +42,39 @@
...
+      <entry>&Version;</entry>
+      <entry>&Description;</entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>8.4.0</entry>
+      <entry>
+       <function>libxml_set_streams_context</function> now throws a
+       <classname>TypeError</classname> when a non-stream resource is passed
```
Comment (Girgias) [inline reference/libxml/functions/libxml-set-streams-context.xml:68]:
```suggestion
       <exceptionname>TypeError</exceptionname> when a non-stream resource is passed
```

File: reference/libxml/setup.xml [hunk 3]
```diff
@@ -9,7 +9,7 @@
   &reftitle.required;
   <para>
    This extension requires <link xlink:href="&url.libxml;">libxml</link> &gt;=
-   2.9.0 as of PHP 8.0. libxml &gt;= 2.6.0 prior to PHP 8.0.
+   2.9.4 as of PHP 8.4. libxml &gt;= 2.9.0 prior to PHP 8.4, and libxml &gt;= 2.6.0 prior to PHP 8.0.
```
Comment (Girgias) [inline reference/libxml/setup.xml:12]: Version should include the patch version
```suggestion
   2.9.4 as of PHP 8.4.0 libxml &gt;= 2.9.0 prior to PHP 8.4.0, and libxml &gt;= 2.6.0 prior to PHP 8.0.0.
```

**Review by Girgias — APPROVED** [review]
Minor nits, but LGTM

File: reference/libxml/functions/libxml-set-streams-context.xml (hunk 1 above)
Comment (nielsdos) [reply reference/libxml/functions/libxml-set-streams-context.xml]: I noticed actually and have fixed this before you got the chance to post this ;)

**Pull request merged**